src/gossip/helpers.zig

```diff
@@ -0,0 +1,53 @@
+const std = @import("std");
+const sig = @import("../sig.zig");
+
+const Pubkey = sig.core.Pubkey;
+const ContactInfo = sig.gossip.ContactInfo;
+const Logger = sig.trace.Logger;
+const Network = sig.cmd.config.Network;
+const GossipService = sig.gossip.GossipService;
+const SocketAddr = sig.net.SocketAddr;
+
+const resolveSocketAddr = sig.net.net.resolveSocketAddr;
+const getMyDataFromIpEcho = sig.cmd.cmd.getMyDataFromIpEcho;
+const getOrInitIdentity = sig.cmd.helpers.getOrInitIdentity;
```

Comment:
It's hard for me to imagine a better place for this other than something like `src/config`.

Reply:
i feel like its fine in cmd/helpers -- its primarily used in binaries/cmd.zig -- config is more confusing than cmd/helpers imo